### PR TITLE
feat: image component added

### PR DESCRIPTION
**Pull request by paarthbajaj**
image-comp

### CODE REVIEW

File: images.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            transform: scale(1);
+            overflow: hidden;
+          "
+          sandbox="allow-scripts allow-same-origin"
+        >
+        </iframe>
+        <h4 class="heading-3">Example of Round Image</h4>
+        <div class="container-box m-radius">
+          <img
+            class="rounded-circle cover-img"
```
Comment (JantuDeb) [inline images.html:77]: You are using `l-radius`  at the top here it is `rounded-circle` .You should be consistent with class names.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes

File: images.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        <a class="link white" href="./index.html">Home</a>
+        <a class="link white" href="./Introduction.html">Components</a>
+      </nav>
+    </header>
+    <div class="main-content flex-row">
+      <aside class="sidebar">
+        <h3 class="sidebar-heading">Components</h3>
+        <ul class="unstyled">
+          <li><a href="./Introduction.html">Intro</a></li>
+          <li><a href="./Avatar.html">Avatar</a></li>
```
Comment (tusharkandpal) [inline images.html:30]: Change `Avatar` to `avatar` just to be consistent with file names.

**Review by tusharkandpal — APPROVED** [review]
Looks good to me ! Just go over what @JantuDeb suggested.

File: images.html (hunk 2 above)
Comment (paarthbajaj) [reply images.html:30]: Sure noted.

File: images.html (hunk 1 above)
Comment (paarthbajaj) [reply images.html:77]: Ok noted. Previously when I have used libraries in other projects, I saw this pattern and it kind of remains in my head. That's why used m-radius, l-radius and for complete circle, I have used rounded-circle.

**Pull request merged**